### PR TITLE
created the first part of the manual via mkdocs

### PR DESCRIPTION
**Pull request by grapesmoker**
This is the first step to putting together a comprehensive manual of how cfgov-refresh operates. I've included instruction on installation and also building/deployment. Next step is front-end stuff.

### CODE REVIEW

**Comment by grapesmoker** [comment]
You can see the compiled form of the manual by doing
```
pip install mkdocs
cd manual
mkdocs serve
```

File: manual/docs/build-and-deploy.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+The following teams and individuals have the necessary access to deploy cf.gov releases:
+
+- Ross Karchner
+- Scott Cranfill
+- Will Barton
+- Bill Higgins
+- Serghei Gorobet
+- The Software Delivery team
+
+### Feature flags
```
Comment (willbarton) [inline manual/docs/build-and-deploy.md]: When I added this section and the Wagtail pages vs Django views section below to the working doc, I envisioned them being separate in their final Markdown form.

File: manual/docs/build-and-deploy.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+{% endif %}
+```
+
+Feature flags are implemented as a Django app in cfgov-refresh/cfgov/flags. 
+
+#### Feature Flag Hygiene
+
+Feature flags should be rare and ephemeral. Changes should be small and frequent, and not big-bang releases, and flags that are no longer used should be cleaned up and removed from code and the database.
+
+### Wagtail pages vs. Django views
```
Comment (willbarton) [inline manual/docs/build-and-deploy.md]: This section generally was a work-in-progress. We can go ahead and add it, but there may be heavy modification necessary. I'd prefer to have some input from @rosskarchner and @richaagarwal and anyone else who knows Wagtail as to whether this is a reasonable starting point or not.

Comment (richaagarwal) [inline manual/docs/build-and-deploy.md]: This seems like a fine starting point for me as we want to encourage content be created via Wagtail going forward. I'm not sure the HelloWorldPage example is a useful one though as, to get the benefits of the CMS, that should be defined via page attributes.

File: manual/docs/build-and-deploy.md (hunk 1 above)
Comment (grapesmoker) [inline manual/docs/build-and-deploy.md]: I'm happy to break this out however people want. Or feel free to move it around if you like. I just took the whole page that was available and turned it into Markdown.

**Comment by grapesmoker** [comment]
closing this in favor of the PR that follows our standard procedure

**Pull request closed without merging**